examples/react/nextjs-swr/codegen.ts

```diff
@@ -1,4 +1,5 @@
 // eslint-disable-next-line import/no-extraneous-dependencies
+// @ts-ignore
```

Comment:
for some reason though, I faced some errors regarding:
```
Type error: Cannot find module '@graphql-codegen/cli' or its corresponding type declarations.

  1 | // eslint-disable-next-line import/no-extraneous-dependencies
> 2 | import { CodegenConfig } from '@graphql-codegen/cli'
```
when running yarn workspace example-react-nextjs-swr run build locally which I skipped with a `// @ts-ignore` for now, do you have any ideas on why is it throwing that?